spec.html

```diff
@@ -31521,7 +31385,7 @@
           1. Let _count_ be max(_final_ - _k_, 0).
           1. Let _defaultConstructor_ be the intrinsic object listed in column one of <emu-xref href="#table-49"></emu-xref> for the value of _O_'s [[TypedArrayName]] internal slot.
           1. Let _C_ be ? SpeciesConstructor(_O_, _defaultConstructor_).
-          1. Let _A_ be ? AllocateTypedArray(_C_, _count_).
+          1. Let _A_ be ? Construct(_C_, «_len_»).
```

Comment:
Typo: len -> count
And AllocateTypedArray ensures a new TypedArray instance is created, this is not the case with Construct.